src/main/scala/com/raphtory/core/config/telemetry/GraphBuilderTelemetry.scala

```diff
@@ -0,0 +1,68 @@
+package com.raphtory.core.config.telemetry
+
+import com.raphtory.core.config.ConfigHandler
+import io.prometheus.client.{Counter, Gauge}
+
+/**
+  * {s}`GraphBuilderTelemetry`
+  *  : Adds metrics for {s}`GraphBuilder` using Prometheus Client
+  *
+  *    Exposes Counter and Gauge stats for tracking number of vertices and edges added and deleted, total partitions created
+  *    by the graph builder
+  *    Statistics are made available on http://localhost:8899 on running tests and can be visualised using Grafana dashboards
+  *
+  *
+  */
+object GraphBuilderTelemetry {
```

Comment:
Not sure if the names here are needed to add in the builderID. Might be an idea so we can see if one is lagging behind etc.

Reply:
I've added a `builder_id` var to this object that could then be set as required.